doc/source/serve/tutorials/vllm-example.md

```diff
@@ -0,0 +1,51 @@
+---
+orphan: true
+---
+
+(serve-vllm-tutorial)=
+
+# Serve a Large Language Model with vLLM
+This example runs a Large Language model with Ray Serve using [vLLM](https://docs.vllm.ai/en/latest/), a popular open-source library for serving LLMs. It uses the [OpenAI Chat Completions API](https://platform.openai.com/docs/guides/text-generation/chat-completions-api) which can be used to easily integrate with other LLM tools. The example also demonstrates how to set-up multi-GPU serving with Ray Serve using placement groups.
+
+To run this example, install the following:
+
+```bash
+pip install "ray[serve]" requests vllm
+```
+
+This example uses the [NousResearch/Meta-Llama-3-8B-Instruct](https://huggingface.co/NousResearch/Meta-Llama-3-8B-Instruct) model. Save the following code to a file named llm.py.
+
+The Serve code is as follows:
```

Comment:
```suggestion
This example uses the [NousResearch/Meta-Llama-3-8B-Instruct](https://huggingface.co/NousResearch/Meta-Llama-3-8B-Instruct) model. Save the following code to a file named llm.py:
```

Reply:
I think there's a way to add a filename to the literalinclude but don't remember the syntax

Reply:
Maybe try this `caption` option :)
https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/code/literalinclude/literalinclude.html#caption